src/Util/Migration/MigrationHelper.php

```diff
@@ -201,15 +201,16 @@ private function languageColumnAlreadyExists(): bool
                 LanguageDefinition::ENTITY_NAME,
                 PackLanguageDefinition::PACK_LANGUAGE_FOREIGN_KEY_STORAGE_NAME,
             ],
-            $sql
+            $sql,
         ));
 
         return (bool) $result->fetchOne();
     }
 
     /**
-     * @return array<mixed>
      * @throws MissingLocalesException
+     *
+     * @return array<mixed>
```

Comment:
Shouldn't this be quite easy to properly type?
You can declare at the top something like
`array<string, string>`